### ISSUE TITLE
ERROR: LoadError: UndefVarError: libkahypar not defined

### ISSUE THREAD

**Issue by Fishermanljh** [kahypar/KaHyPar.jl#26]
Sorry to bother you again, since I am a new comer to Julia. While installing the package on VScode in Windows, I met the following question (for the dependent package: KaHyPar ). I have no idea what is going on, could you please give me some hints on this. Many and many thanks for your help.

Precompiling project...
✗ KaHyPar
0 dependencies successfully precompiled in 1 seconds. 10 already precompiled.
1 dependency errored. To see a full report either run import Pkg; Pkg.precompile() or load the package
Testing Running tests...
ERROR: LoadError: UndefVarError: libkahypar not defined

**Comment by jalving** [comment]
Hi @Fishermanljh. KaHyPar unfortunately does not have Windows binaries. It is my understanding that the KaHyPar build process does not really support cross-compilation. @SebastianSchlag is this still the case? Are there any efforts towards producing a Windows build?

**Comment by SebastianSchlag** [comment]
Hi folks,
yes, support for Windows is still lacking. We had successful Windows builds in the past though, so in principle it should be possible to compile KaHyPar on a Windows machine. Alternatively, using the Windows Subsystem for Linux (and KaHyPar linux builds) could be a viable option.

**Comment by Fishermanljh** [comment]
Many thanks for your kindly reply and help. I will try the Linux system. 

I also wonder whether this package supports the Mac OS ?

**Comment by jalving** [comment]
Yes, we have hosted builds for Mac OS. The M1 architecture should work as well.

**Comment by SebastianSchlag** [comment]
> Yes, we have hosted builds for Mac OS. The M1 architecture should work as well.

Yep, MacOS/M1 should work out of the box.
